Content.Server/Speech/EmotesMenuSystem.cs

```diff
@@ -22,9 +23,16 @@ private void OnPlayEmote(PlayEmoteMessage msg, EntitySessionEventArgs args)
         if (!player.HasValue)
             return;
 
-        if (!_prototypeManager.TryIndex(msg.ProtoId, out var proto) || proto.ChatTriggers.Count == 0)
```

Comment:
Don't just remove this check, match it with the one done in the emote menu.

Reply:
I removed this, because I'm checking for the two separate cases in the if/else right after. It is functionally the same as an early return.